packages/maskbook/src/background-service.ts

```diff
@@ -12,3 +12,4 @@ import './social-network-adaptor' // setup social network providers
 import './extension/background-script/Jobs' // start jobs
 import './plugins/PluginSerivce' // setup plugins
 import './utils/debug/general'
+import './settings/listener'
```

Comment:
Please move this to `packages\maskbook\src\extension\background-script\Jobs\index_hmr.ts` and follow the pattern of `CancelableJob`, thanks.